### ISSUE TITLE
Accessibility & Planning - Dev (FY26 Q3)

### ISSUE THREAD

**Issue by chriswnek** [agrc/planning-queue#864]
### Benefit

The focus this quarter during Sprint 6 is to work through the list of our websites in Popetech and address all accessibility errors.

### Acceptance Criteria

- [ ] The [UGRC Website list spreadsheet](https://docs.google.com/spreadsheets/d/1kOCEp_dxaj2PrrNLfWuP_bPMQvX7GsGtL2bBdAdQhko/edit?usp=sharing) outlines the list of websites and the priority order that they should be worked through.

### Notes

- 

### Risks

- 

### Issue Reference

- #764 
- #813
